test/test_zstar_coordinate.jl

```diff
@@ -300,21 +302,16 @@ end
 
 @testset "TripolarGrid ZStar tracer conservation tests" begin
     for arch in archs
-    	@info "Testing ZStar coordinate with TripolarGrid on $(arch)..."
-
-    	model = make_tripolar_test_model(arch, with_wind=false)
-    	Δt = 2minutes
-    	test_zstar_coordinate(model, 300, Δt)
-    end
-end
-
-@testset "ZStar tracer conservation tests with wind stress" begin
-    for arch in archs
-        @info "Testing ZStar coordinate with TripolarGrid and wind stress on $(arch)..."
-    
-        model = make_tripolar_test_model(arch; with_wind=true)
-        Δt = 2minutes
-        test_zstar_coordinate(model, 300, Δt)
+	for with_wind in (false, true)
+            for with_bathymetry in (false, true)
```

Comment:
I don't think we can use the TripolarGrid without bathymetry (a downside of Tripolar) because of the coordinate singularities

Reply:
oh okay, that explains 

> However, tracer conservation fails at the initial time step for the TripolarGrid without bathymetry (both with and without wind). So this adds more questions than answers...

Reply:
okay, I will roll back